ScreenControl_Unity/Assets/ScreenControl/ServiceUI/Scripts/Configuration/ConfigUI/GlobalSettingsConfigUI.cs

```diff
@@ -195,7 +195,6 @@ protected override void ValidateValues()
             {
                 ConfigManager.InteractionConfig.InteractionType = ScreenControlTypes.InteractionType.HOVER;
             }
-            InteractionManager.Instance.SetActiveInteractions(ConfigManager.InteractionConfig.InteractionType);
```

Comment:
Why aren't we setting this any more? :S

Reply:
The UI application doesn't have an InteractionManager anymore so we don't need to set it, the configuration file is still set and updated (which triggers SetActiveInteractions to be called on the _actual_ service via file watching)

Reply:
Ahhhhhh I see, that's all good then 👍